app/assets/stylesheets/application.scss

```diff
@@ -55,10 +55,16 @@
   box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
 }
 #index-button{
-  margin-left: 10px;
+  margin-left: 20px;
   box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
   margin-bottom: 48px;
 }
+.med-index-button{
+  margin-left: 20px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  margin-bottom: 75px;
+  margin-top: 0px;
+}
 .col-sm-5{
```

Comment:
This is a bit weird to be overriding bootstrap classes. (Generally a big no-no because it defeats the point of using the third party library). Perhaps target the elements using a custom css class and use those

Reply:
Is it the .col-sm-5 tag that shouldn't be used or is it an issue with the box shadow and margins?